src/main/java/com/minecraftabnormals/abnormals_core/common/advancement/modification/AdvancementModificationManager.java

```diff
@@ -101,7 +102,14 @@ private static TargetedAdvancementModifier deserializeModifiers(JsonElement json
 			if (config == null) {
 				throw new JsonParseException("Missing 'config' element!");
 			}
-			advancementModifiers.add(modifier.deserialize(config, conditionArrayParser));
+			boolean conditionsMet = true;
+			if (JSONUtils.hasField(entry, "conditions")) {
+				if (!CraftingHelper.processConditions(JSONUtils.getJsonArray(entry, "conditions"))) {
```

Comment:
I think it'd be best to have this check come before all the other checks in this lamba.
It'd allow advancement modifiers to use the types from another mod conditionally.

Reply:
Yeah that makes sense. I'll commit it tomorrow